piccolo3/server/PiccoloConfig.py

```diff
@@ -51,9 +51,18 @@
 
 [coolbox]
   update_interval = integer(default=5) # update inteval in sec
+  serial_location = string(default="/dev/ttyUSB0")
```

Comment:
I would call it serial_port rather than serial_location

Reply:
Good suggestion, done